aiopolly/utils/strenum.py

```diff
@@ -99,7 +100,13 @@ def __prepare__(mcs, *args, sep: AnyStr = None, converter: Callable[[str], str]
         return super().__prepare__(*args, **kwargs)
 
     def __new__(mcs, cls, bases, class_dict, sep: AnyStr = None, converter: Callable[[str], str] = None):
-        mixin_type, base_enum = mcs._get_mixins_(bases)
+        # In Python 3.8 the signature of 'EnumMeta._get_mixins_' was changed from (bases) to (class_name, bases)
+        # However, 'class_name' is unneccessary and only used to print an exception
+        # So we examine the signature accordingly and pick the right one
+        if len(signature(mcs._get_mixins_).parameters) == 2:  # (class_name, bases)
+            mixin_type, base_enum = mcs._get_mixins_('StrEnumMeta', bases)
```

Comment:
Looks like it expects name of created class, not metaclass name: https://github.com/python/cpython/blob/master/Lib/enum.py#L307
```suggestion
            mixin_type, base_enum = mcs._get_mixins_(cls, bases)
```

Reply:
Yeah, you're right, although it only gets printed during an exception so it has little effect.